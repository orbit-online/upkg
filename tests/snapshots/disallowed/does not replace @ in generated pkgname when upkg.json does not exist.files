[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  with@in-name.tar@7968a50a25bd7e5ca8148dbfb4657ef1147ae8e2c3240597d4936cafb9cd1591
│   │       └── [-rw-r--r--]  empty
│   └── [lrwxrwxrwx]  with@in-name.tar -> .packages/with@in-name.tar@7968a50a25bd7e5ca8148dbfb4657ef1147ae8e2c3240597d4936cafb9cd1591
└── [-rw-r--r--]  upkg.json
