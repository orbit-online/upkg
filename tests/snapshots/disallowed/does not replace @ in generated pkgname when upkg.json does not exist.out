upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/disallowed/with@in-name.tar'
